### PR TITLE
add TC_6

### PR DESCRIPTION
**Pull request by MelisPavol**

### CODE REVIEW

File: src/test/java/SingInTest.java [hunk 1]
```diff
@@ -65,5 +65,15 @@ void test_TC_5_Invalid_password(){
 
         assertEquals("Login failed! Please ensure the username and password are valid.", appointmentPage.getTextLoginFailed());
     }
```
Comment (darinamas) [inline src/test/java/SingInTest.java:72]: Chybi prazdny radek mezi testy

File: src/test/java/SingInTest.java [hunk 2]
```diff
@@ -65,5 +65,15 @@ void test_TC_5_Invalid_password(){
...
+    @Test
+    void test_TC_6_Invalid_test_Empty_textfield_username() {
+        var appointmentPage = homePage
+                .makeAppointmentButton()
+                .enterUserName(TestValues.TEST_INVALIDD_USERNAME_EMPTY)
```
Comment (darinamas) [inline src/test/java/SingInTest.java]: Chyba v INVALIDD nazve promenne :TEST_INVALIDD_USERNAME_EMPTY

**Review by darinamas — CHANGES_REQUESTED** [review]

File: src/test/java/SingInTest.java (hunk 1 above)
Comment (MelisPavol) [reply src/test/java/SingInTest.java:72]: fix

File: src/test/java/SingInTest.java (hunk 2 above)
Comment (MelisPavol) [reply src/test/java/SingInTest.java]: fix

**Review by darinamas — APPROVED** [review]
LGFM

**Pull request merged**